.evergreen/config.yml

```diff
@@ -860,19 +878,38 @@ axes:
           DEPENDENCIES: "lowest"
 
 buildvariants:
-# Test all PHP versions with latest-stable MongoDB and PHPC on Debian
+# Test all PHP versions with latest-stable MongoDB and PHPC, RHEL, and Ubuntu
 - matrix_name: "test-php-versions"
-  matrix_spec: { "os": "debian11", "mongodb-edge-versions": "latest-stable", "php-versions": "*", "driver-versions": "latest-stable" }
+  matrix_spec:
+    os:
+      - debian11
+      - debian10
+      - rhel90
+      # TODO: RHEL 8.x is not working yet
+      # - rhel82-arm64
+      # - rhel80
+      # TODO: RHEL 7.6 fails as it's not able to connect to pecl.php.net to install the library
```

Comment:
If we intend to resolve this, I'd suggest opening a "Build Failure" ticket and referencing it here.

Reply:
While I'm not convinced we need to test on RHEL here, this matrix exposed problems installing extensions via PECL due to SSL issues. I'm not sure how relevant that is, but maybe @remicollet can shed some light on how popular installing via PECL is (and maybe even what the source of the certificate errors is).

Reply:
Do you have details on the SSL error running `pecl install`? He's probably need to see that to provide any insight.

Reply:
Thanks @remicollet for confirming. I'll check with our build team to see what's going on with our build hosts.